<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginPage</class>
 <widget class="QWidget" name="LoginPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="logo_tool_button">
       <property name="styleSheet">
        <string notr="true">QToolButton {
	border-radius: 1px solid transparent;
	background-color: transparent;

	padding: 0px;
	margin: 0px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="EasyVPN.qrc">
         <normaloff>:/assets/app.png</normaloff>:/assets/app.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="title_label">
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;Lucida Sans Unicode&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Easy VPN</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>180</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="login_widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>460</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">.QWidget {
	border-top-left-radius: 30px;
	border-top-right-radius: 30px;
	background-color: rgb(255, 255, 255);
}

</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QLabel" name="username_label">
        <property name="styleSheet">
         <string notr="true">QLabel {
	font: 9pt &quot;Microsoft YaHei&quot;;
	color: rgb(127, 66, 184);
}</string>
        </property>
        <property name="text">
         <string>用户名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="username_line_edit">
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	height: 50px;
	border: 1px solid;
	border-color: rgb(222, 222, 222);
	border-radius: 5px;
	font: 10pt &quot;Microsoft YaHei&quot;;
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>请输入用户名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="password_label">
        <property name="styleSheet">
         <string notr="true">QLabel {
	font: 9pt &quot;Microsoft YaHei&quot;;
	color: rgb(127, 66, 184);
}</string>
        </property>
        <property name="text">
         <string>密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="password_line_edit">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	height: 50px;
	border: 1px solid;
	border-color: rgb(222, 222, 222);
	border-radius: 5px;
	font: 10pt &quot;Microsoft YaHei&quot;;
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>请输入密码</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="connect_push_button">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	height: 50px;
	font: 75 12pt &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
	border: 1px solid transparent;
	border-radius: 8px;
	background-color: rgb(127, 66, 184);
}

QPushButton:hover {
	height: 50px;
	font: 75 12pt &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
	border: 1px solid transparent;
	border-radius: 8px;
	background-color: rgb(155, 101, 205);
}


</string>
        </property>
        <property name="text">
         <string>连接</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="EasyVPN.qrc"/>
 </resources>
 <connections/>
</ui>
